app/modules/ws2812.c

```diff
@@ -51,7 +51,7 @@ static void ICACHE_RAM_ATTR ws2812_write(uint8_t *pixels, uint32_t length) {
   // 0b00000111 => 000111 => [0]111000[1] => 10001110 => 01
   // 0b00110100 => 110100 => [0]001011[1] => 11101000 => 10
   // 0b00000100 => 000100 => [0]001000[1] => 11101110 => 11
-  uint8_t _uartData[4] = { 0b00110111, 0b00000111, 0b00110100, 0b00000100 };
+  static const uint8_t _uartData[4] = { 0b00110111, 0b00000111, 0b00110100, 0b00000100 };
```

Comment:
In fact, without "static const", the array is initialized on the stack from .rodata. With "static const" the code access directly to the .rodata section. The function is then 30 bytes smaller with "static const": good point.

The question is: how slower it is ? It is hard tu measure because everything is interleaved with fifo cache. Which tradeoff is in use in NodeMCU code ? Speed or Free Memory ?